src/Utils.php

```diff
@@ -270,11 +270,14 @@ public static function isHostInNoProxy(string $host, array $noProxyArray): bool
      */
     public static function jsonDecode(string $json, bool $assoc = false, int $depth = 512, int $options = 0)
     {
+        assert($depth >= 1);
+
         $data = \json_decode($json, $assoc, $depth, $options);
         if (\JSON_ERROR_NONE !== \json_last_error()) {
             throw new InvalidArgumentException('json_decode error: ' . \json_last_error_msg());
         }
 
+        /** @var object|array|string|int|float|bool|null */
```

Comment:
@GrahamCampbell Should this also be reverted?

Is it because we already have `@return` in the doc block?